src/main/java/org/javacord/bot/commands/InfoCommand.java

```diff
@@ -0,0 +1,28 @@
+package org.javacord.bot.commands;
+
+import de.btobastian.sdcf4j.Command;
+import de.btobastian.sdcf4j.CommandExecutor;
+import org.javacord.api.DiscordApi;
+import org.javacord.api.Javacord;
+import org.javacord.api.entity.channel.TextChannel;
+import org.javacord.api.entity.message.embed.EmbedBuilder;
+
+public class InfoCommand implements CommandExecutor {
+
+    @Command(aliases = "!info", async = true)
+    public void handleCommand(TextChannel channel) {
+        final DiscordApi api = channel.getApi();
+        EmbedBuilder embed = new EmbedBuilder()
+                .setTitle(api.getYourself().getName() + " - Official Javacord Bot")
+                .setThumbnail(api.getYourself().getAvatar())
+                .setDescription("The official bot for the Javacord Server.\n\n" +
+                        "Powered by Javacord and sdcf4j")
+                .addInlineField("GitHub", "https://github.com/Javacord/Javacord-Bot")
+                .addInlineField("Javacord Version", Javacord.VERSION)
+                .addInlineField("sdcf4j Version", "v1.0.10")
+                .addInlineField("Other Dependencies", "okhttp3, jackson, log4j2");
```

Comment:
I'm not sure it is a good idea to list all dependencies.
For Javacord and sdcf4 it is nice to advertise them.
But I think it might be overkill to list all dependencies.
And if so, then maybe really all and not just some?
But I really doubt this line should be added, what do you think?

Reply:
I have instead opted to link to the `build.gradle`